lib/intercom-rails/script_tag.rb

```diff
@@ -78,6 +82,16 @@ def csp_sha256
       csp_hash
     end
 
+    def find_lead_attributes
+      lead_attributes = IntercomRails.config.user.lead_attributes
+      return {} unless controller.present? && lead_attributes && lead_attributes.size > 0
+
+      # Get custom data. This also allows to retrieve custom data
+      # set via helper function intercom_custom_data
+      custom_data = controller.intercom_custom_data.user.with_indifferent_access
+      custom_data.select {|k, v| lead_attributes.map(&:to_s).include?(k)}
+    end
+
```

Comment:
`find_lead_attributes` also insert stuff when a user in logged-in. How do you feel about that behavior ? Should it just work for leads since there is already another way to insert custom_attr for users ?